modules/ROOT/pages/values-and-types/graph-references.adoc

```diff
@@ -26,13 +26,17 @@ These include:
 [[rules]]
 == Rules
 
-The following rules apply for graph references when using identifiers (static graph references, administration commands) or the `graph.byName` functions:
-
-* Unquoted dots are separators between a composite database and its constituent. 
-For example, `composite.db1` represents the constituent `composite.db1` in the composite database `composite`.
-To refer to a database with a dot (`.`) in its name, quote the graph reference instead: `++`composite.db1`++`.
```

Comment:
while we don't care about the quoting part, it might be good to somewhere at least mention how we handle dots and composite database and their aliases